### PR TITLE
Update `ModelFactory::replaceMusclesWithPathActuators` and add ModelOperator

### PR DESCRIPTION
**Pull request by nickbianco**
### Brief summary of changes

- Include `PathWrap` objects in `ModelFactory::replaceMusclesWithPathActuators`
- Add `ModOpReplaceMusclesWithPathActuators` to ModelOperators.h

### Testing I've completed
- Added test to `testModelProcessor.cpp`

### CHANGELOG.md (choose one)

- updated.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3253)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ v4.4.1
 ======
 
 - Made `Component::getSocketNames` a `const` member method (previously: non-const)
+- Added `ModOpReplaceMusclesWithPathActuators`
```
Comment (aymanhab) [inline CHANGELOG.md:13]: to the list of available model operators in ....

**Review by aymanhab — CHANGES_REQUESTED** [review]
LGTM, just need to clarify the changelog message, and open an issue related to mapping properties between muscles for future consideration.

> Reviewed 5 of 5 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3253#-:-NEh5sUY0Vv-FRtPRJzd:b-wz6nzq)* status: all files reviewed, 2 unresolved discussions (waiting on @nickbianco)

___
*[`OpenSim/Actuators/DeGrooteFregly2016Muscle.cpp` line 966 at r1](https://reviewable.io/reviews/opensim-org/opensim-core/3253#-NEh4bqH6_4bMuXsRu5p:-NEh4bqH6_4bMuXsRu5q:bvente4) ([raw file](https://github.com/opensim-org/opensim-core/blob/780f7f68dd9e79c17e6fdac53024fbd5635eaf27/OpenSim/Actuators/DeGrooteFregly2016Muscle.cpp#L966)):*
> ```cpp
>         // peform muscle-model-specific mappings or throw exception if muscle
>         // not supported
>         if (auto musc = dynamic_cast<Millard2012EquilibriumMuscle*>(
> ```

I understand this is just a comment change but we should consider moving this code elsewhere. As it stands DeGrooteFregly2016 needs to know/understand how all other muscle models work which doesn't scale. Just open an issue for that or let me know and I'll open one. Have you considered using a generic tool that maps property values?

**Comment by nickbianco** [comment]
@aymanhab ready for review. I've created issue #3325 based on your request.

**Review by aymanhab — APPROVED** [review]
Thanks @nickbianco
:lgtm:

> Reviewed 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/opensim-org/opensim-core/3253#-:-NIZ51W9-t8YE5J-rvg2:b-iv6vtn)* status: all files reviewed, 2 unresolved discussions (waiting on @nickbianco)

**Comment by nickbianco** [comment]
Thanks @aymanhab!

**Pull request merged**